.github/workflows/acceptance-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [14.x]
```

Comment:
@simoneb You said "instead" in the issue so I replaced it, instead of adding it. Is that ok?

Reply:
Yes, because these are client packages so we don't need to make sure their build runs on multiple versions of Node, as Node is not their runtime